spec/models/boarding_point_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "rails_helper"
+
+RSpec.describe BoardingPoint, type: :model do
```

Comment:
please check if we can skip this explicit mention of `type`

Reply:
Removed `type: :model` from all the Rspec files. Thanks for the feedback vaia.